server/src/middleware/routes.test.ts

```diff
@@ -334,6 +334,57 @@ describe("Endpoints", () => {
       expect(updatedUser.does_ski).not.toEqual("invalid")
     })
   })
+
+  describe("/users/delete-user", () => {
+    beforeEach(async () => {
+      await createUserData(ADMIN_USER_UID)
+      await createUserData(MEMBER_USER_UID)
+      await createUserData(GUEST_USER_UID)
+    })
+
+    afterEach(async () => {
+      await cleanFirestore()
+    })
+
+    it("should delete the user", async () => {
+      const res = await request
+        .delete("/users/delete-user")
+        .set("Authorization", `Bearer ${adminToken}`)
+        .send({ uid: MEMBER_USER_UID })
+
+      expect(res.status).toEqual(200)
+      const deletedUser = await new UserDataService().getUserData(
+        MEMBER_USER_UID
+      )
+      expect(deletedUser).toEqual(undefined)
+      const unaffectedUser = await new UserDataService().getUserData(
+        GUEST_USER_UID
```

Comment:
we should check if the user is actually deleted in auth, as that will cause problems if they do

note the uid is same for both auth and firestore

Reply:
idk if the `AuthService` already has a method for that but if not can just use the `firebase-admin` package in the tests